src/api/environments.ts

```diff
@@ -105,7 +105,9 @@ export async function getEnvironmentFromSuffix(
 ): Promise<EnvironmentResource> {
   const environments = await listEnvironments(serviceSid, client);
   let foundEnvironments = environments.filter(
-    e => e.domain_suffix === domainSuffix
+    e =>
+      e.domain_suffix === domainSuffix ||
+      (domainSuffix.length === 0 && e.domain_suffix === null)
```

Comment:
If you go with my suggestion to use "production" to signify the suffix-less environment, you would probably want to address it here.

Reply:
See previous comment. People could use `production` as a valid domain suffix so using empty string is the most correct representation of suffix-less while ensuring that the library only supports strings